### ISSUE TITLE
UnusedPrivateMember should consider overloaded operators

### ISSUE THREAD

**Issue by arturbosch** [arturbosch/detekt#1444]
## Expected Behavior of the rule
For following code `minus` should get reported but not `plus`. 
```kotlin
import java.util.Date
class Foo {
    val bla: Date = Date(System.currentTimeMillis()) + 300L
    companion object {
        private operator fun Date.plus(diff: Long): Date = Date(this.time + diff)
        private operator fun Date.minus(diff: Long): Date = Date(this.time - diff)
    }
}
```
## Context
In issue #1354 we stopped looking at operator functions as we do not have type and symbol resolution.
